examples/schema/dataflow.py

```diff
@@ -0,0 +1,78 @@
+import duckdb
```

Comment:
This example is a little dense -- it feels more like a test than an example (enumerating a bunch of DF types). Maybe just a few more explanations + a DAG to help.